python/build_all/linux/build.sh

```diff
@@ -11,12 +11,23 @@ cd $build_root
 [ $? -eq 0 ] || exit $?
 cd $build_root
 
-echo copy iothub_client library to samples folder
-cp ~/cmake/python/src/iothub_client.so ./python/device/samples/iothub_client.so
-echo copy iothub_client_mock library to tests folder
-cp ~/cmake/python/test/iothub_client_mock.so ./python/device/tests/iothub_client_mock.so
+if [[ $* == *--python-3* ]]; then
```

Comment:
Please no new duplicate sample code for Py3, lets rather make existing sample run on both.